node_modules/.package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please do not introduce npm if possible because npm introduces lots of dependencies very quickly.